File: codigos/1 - estrutura sequencial/1002 - Area do Circulo/Main.cs
using System;
using System.Globalization;

// class URI {
    static double PI = 3.14159;

    static string calcular_area(double raio){
        double area = PI * Math.Pow(raio, 2);
        return Math.Round(area, 4).ToString("F4");
    }

    // static void Main(string[] args) { 
        CultureInfo.CurrentCulture = CultureInfo.CreateSpecificCulture("en-US");
        double raio = double.Parse(Console.ReadLine());
        string area = calcular_area(raio);
        Console.WriteLine($"A={area}");
    // }
// }

File: codigos/1 - estrutura sequencial/1002 - Area do Circulo/Main.dart
import 'dart:io';
import 'dart:math';

double PI = 3.14159;

double calcular_area(double raio){
  return PI * pow(raio, 2);
}

main(){
  double raio = double.parse(stdin.readLineSync()!);
  String area = calcular_area(raio).toStringAsFixed(4);
  print("A=$area");
}